packages/react-dom/src/__tests__/ReactUpdates-test.js

```diff
@@ -1064,28 +1223,18 @@ describe('ReactUpdates', () => {
       }
     }
 
-    ReactDOM.render(<App />, document.createElement('div'));
-  });
-
-  it('unstable_batchedUpdates should return value from a callback', () => {
```

Comment:
Deleted, since this is only testing `unstable_batchedUpdates`. The test exists in `ReactLegacyUpdates` now.